DevCycle/Models/Cache.swift

```diff
@@ -40,11 +39,6 @@ class CacheService: CacheServiceProtocol {
         return Cache(config: userConfig, user: dvcUser)
     }
     
-    func save(config: Data) {
```

Comment:
was this not getting used?

Reply:
i have a ticket to save the config locally better